src/webui/www/private/views/preferences.html

```diff
@@ -1600,7 +1604,19 @@
                     if (pref) {
                         // Downloads tab
                         // When adding a torrent
-                        $('createsubfolder_checkbox').setProperty('checked', pref.create_subfolder_enabled);
+                        let index = 0;
+                        switch (pref.torrent_content_layout) {
+                            case "Original":
+                                index = 0;
+                                break;
+                            case "Subfolder":
+                                index = 1;
+                                break;
+                            case "NoSubfolder":
+                                index = 2;
+                                break;
+                        }
+                        $('contentlayout_select').getChildren('option')[index].setAttribute('selected', '');
```

Comment:
I'm not a JavaScript expert, but it uses a different syntax elsewhere:
```js
$('contentLayout').selectedIndex = 1;
```

Reply:
I copied what's in this file
https://github.com/qbittorrent/qBittorrent/blob/a579b4a51999d7148ac7c1f3191bee5e95b6816c/src/webui/www/private/views/preferences.html#L1817